RUN-MIGRATION.md

```
@@ -0,0 +1,164 @@
+# Connect to your database
+psql -h your-database-host -U your-username -d your-database-name
+
+# Run the quick migration
+\i QUICK-ADD-COLUMNS.sql
+
+# Or paste the queries directly:
+```
+
+### Option 2: Copy-Paste Individual Queries
+
+**Step 1: Add Columns**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)